tools/loader/ImageClassifier.cpp

```diff
@@ -159,7 +181,8 @@ int main(int argc, char **argv) {
 
   // Load and process the image data into the data Tensor.
   Tensor data;
-  loadImagesAndPreprocess(inputImageFilenames, &data, imageMode);
+  loadImagesAndPreprocess(inputImageFilenames, &data, imageNormMode,
```

Comment:
I'd actually remove both` imageNormMode` and `imageChannelOrder` from the calling side. I imagine having those as a separate param would be beneficial for testing `loadImagesAndPreprocess` function, but there is no easy way to test that directly.

Reply:
@rdzhabarov I'm not sure to understand your request. Do you propose to remove `imageNormMode` and `imageChannelOrder` parameters and to get directly this information from global variables ?

Reply:
Yes. There's no need to pass them, because they are globally accessible.

Reply:
OK, I did that change, even if I don't think it's a good practice to pass information *implicitly* to functions through globals. In this case, it's a constant global, so it's kind of OK.

Reply:
Well, it's not a regular global variable, it's a flag. AFAIK, there's no other way to define LLVM flag other than global. And if it is defined in the file anyway, why limit ourselves to only using these flags in `main()`?